src/button.js

```diff
@@ -82,6 +84,13 @@ class Button extends Sprite.class {
     button.style = srOnlyStyle;
     button.textContent = this.text;
 
+    /**
+     * If the button is disabled.
+     * @memberof Button
+     * @property {Boolean} disabled
+     */
+    this.disabled = this._dn.disabled = disabled;
```

Comment:
Since we can call the public function here, it would be smaller to do. It will also mean the code isn't duplicated here and in the function.

```suggestion
    if (disabled) { 
      this.disable()
    }
```

Reply:
Hi, I have to modify your suggestion a bit because it doesn't pass the tests:

```js
if (disabled) {
    this.disable();
  } else {
    this.enable();
  }
```

This is because in the initial implementation, `this.disabled` never gets set until `disable()` or `enable()` is called so it remains undefined at the beginning. Do you think that this would be a significant problem for the code size?